index.js

```diff
@@ -1,16 +1,67 @@
-var BigNumber = require('bignumber.js')
-
 var Units = {}
 
 var rawUnits = require('./units.json')
 var units = {}
 
 Object.keys(rawUnits).map(function (unit) {
-  units[unit] = new BigNumber(rawUnits[unit], 10)
+  // drop the leading number 1
+  units[unit] = rawUnits[unit].length - 1
 })
 
 Units.units = rawUnits
 
+// add leading zeroes
+// FIXME: improve
+function zeroes (n) {
```

Comment:
you could replace this with `return Array(n).fill(0).join('')` but i don't know if that is more or less efficient